scripts/diffmazing_cropper.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+#
+# Converts diff images from Cypress into snapshots we can commit to Git.
+
+
+if [ $# -eq 0 ]
+then
+  echo ""
+  echo "Usage"
+  echo "  $0 <diff files to convert>"
+  echo ""
+  echo "Example"
+  echo "  $0 ~/downloads/*.diff.png"
+  echo ""
+  exit -1
+fi
+
+
+has_convert=$(command -v convert)
+if ! [ "$has_convert" ]
+then
+  echo ""
+  echo "Missing dependency."
+  echo "Please install ImageMagick first."
+  echo ""
+  exit -1
+fi
+
+
+
+tmp_dir=/tmp/streamlit_cropped_diffs
```

Comment:
To make this script more portable, we can use `mktemp` command, which honors the platform-dependent $TMDIR environment variable:

```suggestion
tmp_dir="$(mktemp -d -t streamlit_cropped_diffs)"
```

From https://linux.die.net/man/1/mktemp:
> Create a temporary file or directory, safely, and print its name. 

Example: https://code-maven.com/create-temporary-directory-on-linux-using-bash